sdk/python/generative-ai/rag/code_first/README.md

```diff
@@ -6,7 +6,7 @@ Read more about their structure [here](./docs/mlindex.md).
 ## Pre-requisites
 
 0. Install `azure-ai-ml` and `azureml-rag`:
-    - `pip install 'azure-ai-ml==1.10.0a20230825006' --extra-index-url https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/`
+    - `pip install 'azure-ai-ml==1.10'`
```

Comment:
do we need to pin azure-ai-ml to 1.10? That is the latest version anyways.

Reply:
We do need to pin `>=1.10`, I'll modify to do that, any lesser version doesn't handle the new connection types.